src/lib/common/tag.h

```diff
@@ -156,4 +159,6 @@ extern std::string startArray
 */
 extern std::string endArray(const std::string& indent, const std::string& tagName, Format format);
 
+
```

Comment:
unnecessary empty lines  (the three empty lines go *before* the header, not after)